_authors/osci_2023/dizhang.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Di Zhang
+short_name: D
+photo: '/assets/media/authors/di.jpg'
```

Comment:
@pea586 For now, let's remove the photo

```suggestion
```

Reply:
Sure, I have deleted it and pushed again.